app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -108,4 +101,51 @@ def test_user_auth_hash
       OmniAuth::AuthHash.new(info: {email: ""}, extra: {raw_info: {ONELOGIN_JWT_CORE_IDENTITY_HASH_KEY => "test"}})
     end
   end
+
+  def further_education_payments_provider_callback(auth)
+    if DfESignIn.bypass?
+      auth = OmniAuth::AuthHash.new({
```

Comment:
The built in [developer strategy](https://github.com/omniauth/omniauth/blob/master/lib/omniauth/strategies/developer.rb) doesn't allow for setting `extra` fields. I'm considering sending up a PR to fix that but for the time being we just build the specific hash we want here.